.travis/build-and-test.sh

```diff
@@ -4,6 +4,29 @@ set -xe
 # Exit if in format testing mode
 if [ $CHECK_CLANG_FORMAT ]; then exit 0; fi
 
-mkdir _build ; cd _build ; cmake .. -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE
+mkdir _build ; cd _build
+if [ -z $CMAKE_CXX_STANDARD ]
+then
+  cmake .. -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE
+else
+  cmake .. -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE -DCMAKE_CXX_STANDARD=$CMAKE_CXX_STANDARD
+fi
+
 make -j1
 make test
+
+# Run the examples in Release mode
+if [ $CMAKE_BUILD_TYPE == "Release" ]
+then
+  # Examples
+  make -s examples-double_pendulum
```

Comment:
Why do these need to be made separately?!

Reply:
@nim65s's PR have changes it. Now they are added as unittests. Check it out this for further information: https://github.com/loco-3d/crocoddyl/pull/662/files